from PyQt5 import QtCore, QtGui, QtWidgets
from vedio import Vedio_Dialog
from ChangeUI import Change_Dialog
from cutUI import Cut_Dialog
class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(1600, 1400)
        self.mainWindow = MainWindow
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(120, 20, 861, 91))
        font = QtGui.QFont()
        font.setFamily("华文细黑")
        font.setPointSize(22)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setGeometry(QtCore.QRect(120, 170, 841, 691))
        self.frame.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame.setObjectName("frame")
        self.toolButton_7 = QtWidgets.QToolButton(self.frame)
        self.toolButton_7.setGeometry(QtCore.QRect(700, 640, 131, 41))
        self.toolButton_7.setObjectName("toolButton_7")
        self.label_2 = QtWidgets.QLabel(self.frame)
        self.label_2.setGeometry(QtCore.QRect(0, 0, 841, 631))
        self.label_2.setText("")
        self.label_2.setPixmap(QtGui.QPixmap("../../Desktop/7.jpg"))
        self.label_2.setScaledContents(True)
        self.label_2.setObjectName("label_2")
        self.label_9 = QtWidgets.QLabel(self.frame)
        self.label_9.setGeometry(QtCore.QRect(10, 630, 691, 61))
        self.label_9.setObjectName("label_9")
        self.frame_2 = QtWidgets.QFrame(self.centralwidget)
        self.frame_2.setGeometry(QtCore.QRect(570, 920, 391, 401))
        self.frame_2.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_2.setObjectName("frame_2")
        self.toolButton_2 = QtWidgets.QToolButton(self.frame_2)
        self.toolButton_2.setGeometry(QtCore.QRect(250, 350, 131, 41))
        self.toolButton_2.setObjectName("toolButton_2")
        self.label_3 = QtWidgets.QLabel(self.frame_2)
        self.label_3.setGeometry(QtCore.QRect(0, 0, 391, 341))
        self.label_3.setText("")
        self.label_3.setPixmap(QtGui.QPixmap("../../Desktop/1.jpg"))
        self.label_3.setScaledContents(True)
        self.label_3.setObjectName("label_3")
        self.label_19 = QtWidgets.QLabel(self.frame_2)
        self.label_19.setGeometry(QtCore.QRect(10, 355, 231, 31))
        self.label_19.setObjectName("label_19")
        self.frame_3 = QtWidgets.QFrame(self.centralwidget)
        self.frame_3.setGeometry(QtCore.QRect(1020, 170, 461, 301))
        self.frame_3.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_3.setObjectName("frame_3")
        self.toolButton_6 = QtWidgets.QToolButton(self.frame_3)
        self.toolButton_6.setGeometry(QtCore.QRect(320, 250, 131, 41))
        self.toolButton_6.setObjectName("toolButton_6")
        self.label_4 = QtWidgets.QLabel(self.frame_3)
        self.label_4.setGeometry(QtCore.QRect(0, 0, 461, 241))
        self.label_4.setText("")
        self.label_4.setPixmap(QtGui.QPixmap("../../Desktop/2.jpg"))
        self.label_4.setScaledContents(True)
        self.label_4.setObjectName("label_4")
        self.label_21 = QtWidgets.QLabel(self.frame_3)
        self.label_21.setGeometry(QtCore.QRect(10, 250, 301, 41))
        self.label_21.setObjectName("label_21")
        self.frame_4 = QtWidgets.QFrame(self.centralwidget)
        self.frame_4.setGeometry(QtCore.QRect(120, 920, 391, 171))
        self.frame_4.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_4.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_4.setObjectName("frame_4")
        self.toolButton_3 = QtWidgets.QToolButton(self.frame_4)
        self.toolButton_3.setGeometry(QtCore.QRect(260, 120, 121, 41))
        self.toolButton_3.setObjectName("toolButton_3")
        self.label_5 = QtWidgets.QLabel(self.frame_4)
        self.label_5.setGeometry(QtCore.QRect(0, 0, 251, 171))
        self.label_5.setText("")
        self.label_5.setPixmap(QtGui.QPixmap("../../Desktop/0.jpg"))
        self.label_5.setScaledContents(True)
        self.label_5.setObjectName("label_5")
        self.label_10 = QtWidgets.QLabel(self.frame_4)
        self.label_10.setGeometry(QtCore.QRect(290, 30, 81, 31))
        self.label_10.setObjectName("label_10")
        self.label_11 = QtWidgets.QLabel(self.frame_4)
        self.label_11.setGeometry(QtCore.QRect(270, 70, 111, 31))
        self.label_11.setObjectName("label_11")
        self.frame_5 = QtWidgets.QFrame(self.centralwidget)
        self.frame_5.setGeometry(QtCore.QRect(120, 1150, 391, 171))
        self.frame_5.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_5.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_5.setObjectName("frame_5")
        self.toolButton_8 = QtWidgets.QToolButton(self.frame_5)
        self.toolButton_8.setGeometry(QtCore.QRect(260, 120, 121, 41))
        self.toolButton_8.setObjectName("toolButton_8")
        self.label_6 = QtWidgets.QLabel(self.frame_5)
        self.label_6.setGeometry(QtCore.QRect(0, 0, 251, 171))
        self.label_6.setText("")
        self.label_6.setPixmap(QtGui.QPixmap("../../Desktop/5.jpg"))
        self.label_6.setScaledContents(True)
        self.label_6.setObjectName("label_6")
        self.label_14 = QtWidgets.QLabel(self.frame_5)
        self.label_14.setGeometry(QtCore.QRect(260, 25, 131, 41))
        self.label_14.setObjectName("label_14")
        self.label_15 = QtWidgets.QLabel(self.frame_5)
        self.label_15.setGeometry(QtCore.QRect(260, 75, 121, 31))
        self.label_15.setObjectName("label_15")
        self.frame_6 = QtWidgets.QFrame(self.centralwidget)
        self.frame_6.setGeometry(QtCore.QRect(1020, 920, 461, 401))
        self.frame_6.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_6.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_6.setObjectName("frame_6")
        self.toolButton = QtWidgets.QToolButton(self.frame_6)
        self.toolButton.setGeometry(QtCore.QRect(320, 350, 131, 41))
        self.toolButton.setObjectName("toolButton")
        self.label_7 = QtWidgets.QLabel(self.frame_6)
        self.label_7.setGeometry(QtCore.QRect(0, 0, 461, 341))
        self.label_7.setText("")
        self.label_7.setPixmap(QtGui.QPixmap("../../Desktop/30.jpg"))
        self.label_7.setScaledContents(True)
        self.label_7.setObjectName("label_7")
        self.label_17 = QtWidgets.QLabel(self.frame_6)
        self.label_17.setGeometry(QtCore.QRect(10, 349, 311, 41))
        self.label_17.setObjectName("label_17")
        self.frame_7 = QtWidgets.QFrame(self.centralwidget)
        self.frame_7.setGeometry(QtCore.QRect(1020, 530, 461, 331))
        self.frame_7.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_7.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_7.setObjectName("frame_7")
        self.toolButton_5 = QtWidgets.QToolButton(self.frame_7)
        self.toolButton_5.setGeometry(QtCore.QRect(320, 280, 131, 41))
        self.toolButton_5.setObjectName("toolButton_5")
        self.label_8 = QtWidgets.QLabel(self.frame_7)
        self.label_8.setGeometry(QtCore.QRect(0, 0, 461, 271))
        self.label_8.setText("")
        self.label_8.setPixmap(QtGui.QPixmap("../../Desktop/9.jpg"))
        self.label_8.setScaledContents(True)
        self.label_8.setObjectName("label_8")
        self.label_16 = QtWidgets.QLabel(self.frame_7)
        self.label_16.setGeometry(QtCore.QRect(10, 280, 301, 41))
        self.label_16.setObjectName("label_16")
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 1600, 22))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def changeUI(self):
        self.mainWindow.hide()
        MainDialog = QtWidgets.QDialog() # 创建一个主窗体（必须要有一个主窗体）
        myDialog = Change_Dialog()  # 创建对话框
        myDialog.setupUi(MainDialog)
        MainDialog.show()
        MainDialog.exec_()
        self.mainWindow.show()

    def cutUI(self):
        self.mainWindow.hide()
        MainDialog = QtWidgets.QDialog() # 创建一个主窗体（必须要有一个主窗体）
        myDialog = Cut_Dialog()  # 创建对话框
        myDialog.setupUi(MainDialog)
        MainDialog.show()
        MainDialog.exec_()
        self.mainWindow.show()

    def vedioUI(self):
        self.mainWindow.hide()
        MainDialog = QtWidgets.QDialog()  # 创建一个主窗体（必须要有一个主窗体）
        myDialog = Vedio_Dialog()  # 创建对话框
        myDialog.setupUi(MainDialog)
        MainDialog.show()
        MainDialog.exec_()
        self.mainWindow.show()

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.label.setText(_translate("MainWindow", "欢迎来到我的VLOG"))
        self.toolButton_7.setText(_translate("MainWindow", "一键生成"))
        self.label_9.setText(_translate("MainWindow", "开始自己的VLOG制作之旅，一键获得自己的炫酷VLOG。"))
        self.toolButton_2.setText(_translate("MainWindow", "视频分割"))
        self.label_19.setText(_translate("MainWindow", "长视频分解成短视频"))
        self.toolButton_6.setText(_translate("MainWindow", "视频剪辑"))
        self.label_21.setText(_translate("MainWindow", "输入起点和终点，剪辑。"))
        self.toolButton_3.setText(_translate("MainWindow", "录屏一下"))
        self.label_10.setText(_translate("MainWindow", "录屏"))
        self.label_11.setText(_translate("MainWindow", "合成音频"))
        self.toolButton_8.setText(_translate("MainWindow", "一键改变"))
        self.label_14.setText(_translate("MainWindow", "一个按钮"))
        self.label_15.setText(_translate("MainWindow", "图片格式化"))
        self.toolButton.setText(_translate("MainWindow", "字幕生成"))
        self.label_17.setText(_translate("MainWindow", "为视频文件，生成字幕。"))
        self.toolButton_5.setText(_translate("MainWindow", "格式转换"))
        self.label_16.setText(_translate("MainWindow", "视频、音频格式转换。"))
        self.toolButton_5.clicked.connect(self.changeUI)
        self.toolButton_3.clicked.connect(self.vedioUI)
        self.toolButton_2.clicked.connect(self.cutUI)




